
uint8  command_type
uint8  START = 1
uint8  STOP = 2
uint8  SUSPEND = 3
uint8  CONFIRM_STOP=4

string path_file_name
uint8  path_type
uint8  VERTEX_TYPE=100
uint8  CURVE_TYPE =101

float32 speed

---

uint8 SUCCESS = 0
uint8 PATH_FILE_NOT_EXIST = 1
uint8 FAIL = 2
uint8 PATH_FILE_ERROR = 3

uint8 success

